care/facility/migrations/0346_add_recommend_discharge_action.py

```diff
@@ -0,0 +1,67 @@
+from django.db import migrations, models
+
+
+def update_recommend_discharge(apps, schema_editor):
+    patient_model = apps.get_model("facility", "PatientRegistration")
+    patient_objs = []
+    for patient in patient_model.objects.all():
+        if (
+            patient.last_consultation
+            and patient.last_consultation.last_daily_round
+            and patient.last_consultation.last_daily_round.recommend_discharge
+        ):
+            patient.action = 90
+            patient_objs.append(patient)
+    patient_model.objects.bulk_update(patient_objs, ["action"])
```

Comment:
We could simply do the following instead right?
```suggestion
    PatientRegistration.objects.filter(last_consultation__last_daily_round__recommend_discharge=True).update(action=90)
```